include/boost/json/string.hpp

```diff
@@ -1799,29 +2241,97 @@ class string
 
     //------------------------------------------------------
 
+    /** Append a character.
+        
+        Appends a character to the end of the string.
+
+        @par Exception Safety
+
+        Strong guarantee.
+        
+        @param ch The character to append.
+        
+        @throw std::length_error `size() + 1 > max_size()`
+    */
     BOOST_JSON_DECL
     void
     push_back(char ch);
 
+    /** Remove a character.
+        
+        Removes a character from the end of the string.
+
+        @par Precondition
+
+        `not empty()`
+    */
     BOOST_JSON_DECL
     void
     pop_back();
 
     //------------------------------------------------------
 
+    /** Append characters to the string.
+        
+        Appends `count` copies of `ch` to the end of the string.
+
+        @par Exception Safety
+
+        Strong guarantee.
+
+        @return `*this`
+
+        @param count The number of characters to append.
+        @param ch The character to append.
+
+        @throw std::length_error `size() + count > max_size()`
+    */
     BOOST_JSON_DECL
     string&
     append(
         std::size_t count,
         char ch);
 
+    /** Append a another string to the string.
+
+        Appends `s` the end of the string.
+
+        @par Exception Safety
+
+        Strong guarantee.
+
+        @return `*this`
+
+        @param s The string to append.
+
+        @throw std::length_error `size() + s.size() > max_size()`
+    */
     string&
     append(string const& s)
     {
         return append(
             s.data(), s.size());
     }
 
+    /** Append a string to the string.
+
+        Appends a string, obtained by `s.substr(pos, count)`
+        to the end of the string.
+
+        @par Exception Safety
+
+        Strong guarantee.
+
+        @return `*this`
+
+        @param s The string to append.
+        @param pos The position from which to begin the substring.
+        @param count The number of characters to append.
+        The default argument for this parameter is `npos`.
```

Comment:
Actually I think you can write `@ref npos` instead of `npos` and Doxygen will link it to the declaration of `string::npos` :)

Reply:
Will do!